src/modules/map/components/LocationPopup.vue

```diff
@@ -98,6 +101,13 @@ watch(currentLang, () => {
 })
 watch(() => route.query, updateShareLink)
 
+watch(showEmbedSharing, () => {
+    if(showEmbedSharing.value)
+    {
+        updateShareLink()
+    }
+})
```

Comment:
Code formatting is not correct, run `npm run format`. This PR passed CI due tohttps://github.com/geoadmin/infra-terraform-bgdi-builder/pull/176 which has been applied but not #588.